content/workshops/modern-css.md

```diff
@@ -0,0 +1,191 @@
+---
+feature: workshop
```

Comment:
@mirisuzanne Does this do anything until we revert #735 (and parts of #736 if we want)? Or maybe we want to feature our own April workshop?

Reply:
Oh, yeah like it doesn't do anything currently. I just moved it over from the previous one.

Reply:
I guess the question is whether we want to feature a workshop, or which one we want to feature next? But for now this doesn't hurt anything.